src/agenda/index.js

```diff
@@ -34,7 +29,7 @@ export default class AgendaView extends Component {
     ...CalendarList.propTypes,
     ...ReservationList.propTypes,
     /** agenda container style */
-    style: viewPropTypes.style,
+    style: PropTypes.object,
```

Comment:
What if I pass style as an array? or StyleSheet ref?

Reply:
Thanks for lookong into this. I forgot about arrays of style objects, those would fail the checks. Not sure about style refs, thought they were objects.

Tricky stuff, and not a lot to gain anyways. Would you be ok with `PropTypes.any`?

Reply:
I have no probblem with 'any' here but isn't it better to limit to `PropTypes.oneOfType([PropTypes.object, PropTypes.array, PropTypes.number])`?

Reply:
I updated the PR with your suggestion.